OpenEmu/OEGameInfoHelper.m

```diff
@@ -371,6 +371,16 @@ - (NSDictionary * _Nullable)gameInfoWithDictionary:(NSDictionary *)gameInfo {
             return nil;
         }
 
+        // Remove "-REV*" and "-DISC*" from the serial before database look up
+        if ([key isEqualToString:@"romSerial"]) {
+            if  ([value containsString:@"-REV"]) {
+                value = [value substringWithRange:NSMakeRange(0, [value rangeOfString:@"-REV"].location)];
+            }
+            if ([value containsString:@"-DISC"]) {
+                value = [value substringWithRange:NSMakeRange(0, [value rangeOfString:@"-DISC"].location)];
+            }
+        }
+
```

Comment:
As a workaround, I added -REV and -DISC to GameCube game serials in the database that needed them, because otherwise I cannot uniquely identify them. For example, a "Disc 1" and "Disc 2" of a game have the exact same serial in their header. So I dunno why we'd want to remove them here.

Reply:
Oops. You're correct I messed up when fixing the CISO serial lookup.

I've now left only the CISO fix in this commit.